examples/axum/src/axum_ructe.rs

```diff
@@ -0,0 +1,29 @@
+use axum::response::{Html, IntoResponse};
+
+macro_rules! render {
```

Comment:
Cool macro!  Simple enough, rather different from what the other examples do, but I like it!

Reply:
It's actually the same macro as used in the actix example. I didn't investigate further but it seemed the actix example did duplicate one branch in the macro. Here is the macro from actix:

```
macro_rules! render {
    ($template:path) => (Render(|o| $template(o)));
    ($template:path, $($arg:expr),*) => {{
        use actix_ructe::Render;
        Render(|o| $template(o, $($arg),*))
    }};
    ($template:path, $($arg:expr),* ,) => {{
        use actix_ructe::Render;
        Render(|o| $template(o, $($arg),*))
    }};
}
```

I'm not too deep into Rust macros so I don't know what the trailing `,` in tha last branch of the macro changes in comparison to the second branch but it works.

Reply:
Ah, I see. I think I found a less redundant way by making the trailing comma optional. See cf68d54